packages/atomic/cypress/integration/facets-v1/facet/facet.cypress.ts

```diff
@@ -106,69 +109,325 @@ describe('Facet v1 Test Suites', () => {
         });
       });
     });
+  });
 
-    describe('when selecting the "Show more" button', () => {
-      function setupSelectShowMore() {
-        setupWithCheckboxValues();
-        FacetSelectors.showMoreButton().click();
+  describe('with link values', () => {
+    function setupWithLinkValues() {
+      new TestFixture()
+        .with(addFacet({field, label, 'display-values-as': 'link'}))
+        .init();
+      cy.wait(TestFixture.interceptAliases.UA);
```

Comment:
Is that `cy.wait` something we should add to `TestFixture.init()` ? When "auto trigger first search" is enabled, at least.